src/pages/Club/AllClubs/Clubs.jsx

```diff
@@ -9,31 +9,64 @@ import useSWR from "swr";
 import fetcher from "../../../utils/Fetcher";
 import { clubEndpoints } from "../../../assets/data/ApiEndpoints";
 import ComponentHelmet from "../../../utils/ComponentHelmet";
+import Feedback from "../../../components/Feedback system/Feedback";
+import Modal from "../../../components/Modal/Modal";
 
 const Clubs = () => {
   const { data: clubs } = useSWR(clubEndpoints.all, fetcher);
 
   useEffect(() => {
     window.scrollTo(0, 0);
   }, []);
+  const [review, setReview] = useState(false);
+  if (review) {
+    return (
+      <>
+        <ComponentHelmet type="Clubs" />
+        <Navbar />
+        <button onClick={() => setReview(true)}>Share Feedback</button>
```

Comment:
Please use our [button component](https://github.com/milancommunity/Milan/blob/main/src/components/Button/GlobalButton/Button.jsx) instead of a new button. That way the styles will not look out of place.

![image](https://github.com/milancommunity/Milan/assets/72851613/94181781-7a8d-4b04-a0c0-eef7e6cb16c1)